src/Sentry.Unity/SentryUnity.cs

```diff
@@ -32,7 +32,7 @@ public static void Init(SentryUnityOptions unitySentryOptions)
             // Uses the game `version` as Release unless the user defined one via the Options
             if (unitySentryOptions.Release == null)
             {
-                unitySentryOptions.Release = Application.version;
+                unitySentryOptions.Release = Application.productName + "@" + Application.version;

```

Comment:
Lets go with interpolation:

```suggestion
                unitySentryOptions.Release = $"{Application.productName}@{Application.version}";

```